docs/docs/tutorial/creating-a-source-plugin/part-0/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "Part 0: Introduction and Prerequisites"
+---
+
+import { LinkButton } from "gatsby-interface"
+import { MdArrowForward } from "react-icons/md"
+
+## Introduction
+
+Welcome to the "Creating a Gatsby source plugin" tutorial. We're excited you're here!
+
+In this tutorial you'll learn how to create a Gatsby source plugin that is following best practices, using Gatsby's latest features, and ultimately is how the Gatsby team wants you to build source plugins. By following our instructions your plugin will have great developer experience for you, and great reliability and usability for your users.
+
+Source plugins are an integral part to Gatsby's [GraphQL data layer](/docs/reference/graphql-data-layer/) as they are responsible for bringing data into that layer. They play a key role in hiding complex pieces of software (e.g. fetching and normalizing data from a remote API) behind an easy-to-use plugin. This also means that they have to be reliable, performant, and use best practices behind the scenes.
+
+There is already a wealth of [existing source plugins](/plugins?=gatsby-source) in the Gatsby ecosystem you can use, but there may come a time when you want to create your own source plugin that doesn't already exist. If that sounds like a situation you've found yourself in, this tutorial is for you! Otherwise if you're curious to learn more about how Gatsby's source plugins work, you'll also find useful information here.
+
+<Announcement>
+
+This tutorial is indended to be an **additive guide**. Each part builds up on each other so we recommend you going through it from start to finish. However, if you want to revisit a specific topic each part should have all the necessary information on its own to be informative.
+
+You'll have a functioning source plugin at the end of part 2. The later parts are improving the plugin, adding functionalities, and explaining advanced topics. So while you can already publish what you have after part 2, we highly recommend going through all parts to learn how you can further improve your plugin.
+
+</Announcement>
+
+By the end of this part of the tutorial, you will:
+
+- Know what background knowledge is assumed throughout the tutorial
+- Understand what a source plugin is and why you would want to create one
```

Comment:
Similar to the comment above, understand can be hard to measure here's a few suggestions of action verbs:

```suggestion
- Clarify what a source plugin is and why you would want to create one
```
```suggestion
- Describe what a source plugin is and why you would want to create one
```
```suggestion
- Review what a source plugin is and why you would want to create one
```